src/sonic-config-engine/minigraph.py

```diff
@@ -1848,10 +1851,16 @@ def parse_device_desc_xml(filename):
     if lo_prefix_v6:
         results['LOOPBACK_INTERFACE'] = {('lo_v6', lo_prefix_v6): {}}
 
-    mgmt_intf = {}
-    mgmtipn = ipaddress.ip_network(UNICODE_TYPE(mgmt_prefix), False)
-    gwaddr = ipaddress.ip_address((next(mgmtipn.hosts())))
-    results['MGMT_INTERFACE'] = {('eth0', mgmt_prefix): {'gwaddr': gwaddr}}
+    results['MGMT_INTERFACE'] = {}
+    if mgmt_prefix and mgmt_prefix != '0.0.0.0/0':
+        mgmtipn = ipaddress.ip_network(UNICODE_TYPE(mgmt_prefix), False)
+        gwaddr = ipaddress.ip_address((next(mgmtipn.hosts())))
+        results['MGMT_INTERFACE'].update({('eth0', mgmt_prefix): {'gwaddr': gwaddr}})
+
+    if mgmt_prefix_v6 and mgmt_prefix_v6 != '::/0':
```

Comment:
>mgmt_prefix_v6 != '::/0' [](http://example.com/codeflow?start=26&length=24)

Do we really need to check with all zero IPv4/IPv6? If yes, compare it by ipaddress class, not by string.

Reply:
Yes, minigraph will provide default value (all zero addresses). I will update with ipn comparison